### DISCUSSION TITLE
ScriptReference/MonoBehaviour.Awake

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#609] [Notes]
# ScriptReference/MonoBehaviour.Awake

https://docs.unity3d.com/ScriptReference/MonoBehaviour.Awake.html

**Comment by magnoahlia** [comment]
> Awake is called when an enabled script instance is being loaded.

Per [this issue](https://issuetracker.unity3d.com/issues/disabled-scripts-still-call-awake-function) Awake is called even if the script is disabled by design.
